[INFO 1637204178.428130000] [surge_controller_node]: Stopped with user interrupt (main() at C:\TCC-Autonomous-Ship\src\main_ws\src\install\Lib\site-packages\path_following\surge_controller.py:196)
